tests/shared/src/test/scala/org/http4s/Ipv4AddressSuite.scala

```diff
@@ -36,7 +37,8 @@ class Ipv4AddressSuite extends Http4sSuite {
   if (Platform.isJvm)
     test("fromInet4Address should round trip with toInet4Address") {
       forAll { (ipv4: Ipv4Address) =>
-        assert(Ipv4Address.fromInet4Address(ipv4.toInet4Address) == ipv4)
+        assert(Ipv4Address.fromInet4Address(ipv4.toInet4Address) == ipv4): @nowarn(
+          "cat=deprecation")
```

Comment:
Or else, we should just remove these tests. If we address the deprecation then we are just testing if ip4s works :)

Reply:
Forgive me, I just deleted them after getting into a twist of `nowarn` and `nowarn` doesn't suppress warnings between platforms.